src/store/ConfigStore.ts

```diff
@@ -0,0 +1,25 @@
+import { createStore, applyMiddleware, compose } from 'redux';
```

Comment:
One more time, I'd love me some comments about the purpose of these new, seemingly infra-level, files.

Reply:
This separates the Redux configuration from App.tsx. A single-purpose file to configure redux (this should also help avoid the rebasing issues). Starting to make everything smaller, more independent.